com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/errors.py

```diff
@@ -0,0 +1,2 @@
+class ViewNotFoundError(Exception):
+    """Raised when the SWS view element cannot be located."""
```

Comment:
What's a 'SWS view element'?

Is it just the view cannot be located, or that no such view currently exists for a job/instance?